.prettierignore

```diff
@@ -3,3 +3,5 @@ pnpm-lock.yaml
 apps/homepage/_layouts/compress.html
 apps/homepage/vendor
 apps/homepage/_site
+packages/jsonresume-theme-full
+packages/jsonresume-theme-spartacus
```

Comment:
I do not think that ignoring packages for linting is a good approach. We should fix the linting problems in any new code before we merge. That is usually doable quite easily if we take a little more time and care. I do not see the need for haste. If people who use these themes are impatient, they can fix this stuff immediately and open a PR either here or in the upstream repos.

If really we say: We do not want to fix these codebases we should at least use [`esplint`](https://www.npmjs.com/package/esplint) instead of ignoring code altogether. This will make sure that any new additions to these folders will have to adhere to our style guidelines without enforcing the fixing of the legacy problems.